src/index.ts

```diff
@@ -1,56 +1,298 @@
-import {Business} from '@typings/types'
-import { typeDefs as schema } from '@typings/schema'
+
+import {PrismaClient, Prisma} from '@prisma/client'
+// may not need to import prisma - but idk how to make
+// sure input is valid.
```

Comment:
This comment should be removed

Reply:
I can remove it, but what comments are allowed in the file? Aren't they required for helping to explain code?